rpcs3/Emu/RSX/VK/VKGSRender.cpp

```diff
@@ -2912,16 +2912,58 @@ void VKGSRender::begin_conditional_rendering(const std::vector<rsx::reports::occ
 	{
 		// We'll need to do some result aggregation using a compute shader.
 		auto scratch = vk::get_scratch_buffer(*m_current_command_buffer, num_hw_queries * 4);
+
+		// Range latching. Because of how the query pool manages allocations using a stack, we get an inverse sequential set of handles/indices that we can easily group together.
+		// This drastically boosts performance on some drivers like the NVIDIA proprietary one that seems to have a rather high cost for every individual query transer command.
+		std::pair<u32, u32> query_range = { umax, 0 };
```

Comment:
I think this entire section would be much easier to read if you just used two individual variables with proper names

Reply:
The heurestics would be worse of with separate vars due to their linked nature (e.g only checking one of them to infer the value on the other). Changed to an inline struct instead to split the difference.